app/views/admin/search/_purchases.html.erb

```diff
@@ -28,6 +28,14 @@
                 <li><%= state %></li>
               <% end %>
             </ul>
+            <div class="text-sm">
+              <ul class="inline">
+                <% if purchase.purchase_refund_policy.present? %>
+                  <li>Refund policy: <%= purchase.purchase_refund_policy.title %></li>
```

Comment:
Maybe if the refund policy doesn't match the current purchase.product.refund_policy, show that?